Tests/ReactiveSwiftTests/SchedulerSpec.swift

```diff
@@ -6,12 +6,25 @@
 //  Copyright (c) 2014 GitHub. All rights reserved.
 //
 
+import Dispatch
 import Foundation
+
 import Nimble
 import Quick
 @testable
 import ReactiveSwift
 
+#if os(Linux)
```

Comment:
This is an implementation detail of the tests, not the actual tests. For that reason, could you move it to the bottom of the file?